raiden-dapp/src/components/PlannedUdcWithdrawalInformation.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <div
+    class="planned-udc-withdrawal-information d-flex justify-center align-center px-2"
+    :style="{ color, backgroundColor }"
+  >
+    <template v-if="noWithdrawInProgress">
```

Comment:
Just out of curiosity, what are the benefits of using the `template` element instead of a, for example, `div`?

Reply:
A `v-if` and according `v-else` directive can be only attached to one element in the template. So this basically adds the directive to all its childs, but itself is invisible. So I do not need to repeat if for each child, besides that I can't assign a `v-else` multiple times. The difference to a `div` is, that the directive gets only applied to the div itself. The div is its own object in the DOM and gets rendered as that. So the whole flow and DOM looks different.

```html
<template>
   <span>a</span>
   <img alt="b" />
</template>
```
becomes
```html
<span>a</span>
<img alt="b" />
```

But 
```html
<div>
   <span>a</span>
   <img alt="b" />
</div>
```
remains as it is.

This becomes relevant, because in the if-else scenario, I can define at the parent component the flex flow and style, no matter which case is true. If I put them into divs, the flow gets destroyed and I would need to repeat it or make it reusable.

Sorry for the long answer. But did it become clear?

Reply:
Ok nice, was not aware of that! 👍